### DISCUSSION TITLE
Fasttrack repo error in spirallinux.list

### DISCUSSION THREAD

**Discussion by bin-linux** [SpiralLinux/SpiralLinux-project#55] [General]
Using Spiral Plasma KDE
Just a minor glitch.
After installation the entries for Fasttrack in spirallinux.list cause a little mess.

```
Err:1 https://fasttrack.debian.net/debian-fasttrack bullseye-fasttrack InRelease
  Something wicked happened resolving 'fasttrack.debian.net:https' (-5 - No address associated with hostname)
Err:2 https://fasttrack.debian.net/debian-fasttrack bullseye-backports-staging InRelease
  Something wicked happened resolving 'fasttrack.debian.net:https' (-5 - No address associated with hostname
```

From spirallinux.list:-

```
# Needed for VirtualBox in Debian Stable
deb https://fasttrack.debian.net/debian-fasttrack/ bullseye-fasttrack main contrib non-free  #Bullseye Fast Track (for VirtualBox)
deb https://fasttrack.debian.net/debian-fasttrack/ bullseye-backports-staging main contrib non-free  #Bullseye Fast Track Staging
```

Commenting out the 'for Virtualbox' line makes the error go away - if it shouldn't be there after install perhaps it would help if it was removed as part of the installation?

**Comment by geckolinux** [comment]
Hi there, thanks for the report. I've seen that error from time to time in video reviews that users have posted about SpiralLinux, but it appears to just be a random network error. For example, I just now tested it and it worked fine:

![Screenshot from 2022-07-19 08-04-46](https://user-images.githubusercontent.com/2692138/179757636-be86132d-d58a-413b-8e8c-8c8b6c669e20.png)

As mentioned in the comment, that repo is necessary for VirtualBox support, as SpiralLinux is the only Debian Stable option to include support for it out-of-the-box. Of course, if you're not using VirtualBox then you can disable that repository with Synaptic.

Now that I look closer at it however, I see that the FastTrack repo is the only one using `https`, so I'll look into changing it to a `http` link in future releases, which will probably prevent that random network error.

**Reply by bin-linux** [reply]
It is puzzling though that I can boot off live media and it is fine. Install, reboot and and it errors.
It may be too simplistic but I did try it with http rather than https - no difference.

However, are you saying that repo is needed in order to install VirtualBox on a Spiral installed system, or, that it is needed to be able to run Spiral 'properly' in a VM?

Thanks

**Reply by geckolinux** [reply]
> However, are you saying that repo is needed in order to install VirtualBox on a Spiral installed system, or, that it is needed to be able to run Spiral 'properly' in a VM?

Both.

> It is puzzling though that I can boot off live media and it is fine. Install, reboot and and it errors.

It's really just random I think. Since it's not a commonly used repo the Debian project probably hasn't received reports of this intermittence.

**Reply by bin-linux** [reply]
OK - thanks for the info.
